src/commands/deploy/getDeployNode.ts

```diff
@@ -41,7 +41,8 @@ export async function getDeployNode(context: IDeployContext, target: Uri | strin
         const newNodes: SiteTreeItem[] = [];
         const disposable: Disposable = ext.tree.onTreeItemCreate((newNode: SiteTreeItem) => { newNodes.push(newNode); });
         try {
-            node = await ext.tree.showTreeItemPicker<SiteTreeItem>([WebAppTreeItem.contextValue, TrialAppTreeItem.contextValue], context);
+            const contextValues: string[] = isTargetNewWebApp ? [WebAppTreeItem.contextValue] : [WebAppTreeItem.contextValue, TrialAppTreeItem.contextValue];
```

Comment:
Can you explain the reason behind this check?

Reply:
I should have made a comment, but I think at the time of writing that check it made complete sense 😅 (as always). But I think this check is now obsolete since we are calling `createWebApp` before calling `deploy` in `transferTrialApp`.

I will remove it.